.github/workflows/build_and_test_dbt.yaml

```diff
@@ -81,7 +95,8 @@ jobs:
         working-directory: ${{ env.PROJECT_DIR }}
         shell: bash
 
-      - name: Move dbt manifest directory to update cache
-        run: mv "$TARGET_DIR" "$STATE_DIR"
-        working-directory: ${{ env.PROJECT_DIR }}
-        shell: bash
```

Comment:
[The error in question](https://github.com/ccao-data/data-architecture/actions/runs/5906036003/job/16021252615):

```
Run mv "$TARGET_DIR" "$STATE_DIR"
mv: cannot move 'target' to 'state/target': Directory not empty
Error: Process completed with exit code 1.
```

Saving the cache explicitly with the `actions/cache/save` action resolves this problem.